server/apiserver/apiserver.go

```diff
@@ -156,17 +156,13 @@ func (api *ApiServer) HTTPUpdatePixel(w http.ResponseWriter, req *http.Request)
 func (api *ApiServer) CallUpdatePixel(x int, y int, r int, g int, b int, userID string) []byte {
 	fmt.Println("\nWithin UpdatePixel")
 
-	// TODO verify that the user is able to update a pizel with the User Data Team
-	userVerification := true
-	if !userVerification {
+	userVerification := api.validateUser(userID)
+	if userVerification == 401 || userVerification == 429 {
 		// User verification failed
-
 		log.Println(fmt.Sprintf("USER %s failed authentication", userID))
-		// TODO return the appropriate failure message
-		imageMsg := "FAILURE. TODO make this properly formatted"
 
 		// send message back to the client saying it's been updated or if it failed
```

Comment:
This comment is misleading. It seems to me this is only sent on failure.

Reply:
Message type is changed to `VerificationFailMsg` to indicate failure.